demos/scripts/robot_kinematics_control_loop.cpp

```diff
@@ -0,0 +1,88 @@
+#include "state_representation/space/cartesian/CartesianPose.hpp"
```

Comment:
these should all actually be system includes `<>`, since they are not local to this demo project.

Additionally, the inbuilt libraries (chromo, thread) should come first